qemu/tests/cfg/multi_vms_with_stress.cfg

```diff
@@ -15,9 +15,12 @@
     pre_command = "mkdir -p ${stress_inst_dir}"
     post_command = "killall -9 stress; rm -rf ${stress_inst_dir}"
     ovmf:
-        smp = 1
-        vcpu_maxcpus = 1
-        vcpu_sockets = 1
-        vcpu_dies = 1
-        vcpu_cores = 1
-        vcpu_threads = 1
+        vms = "avocado-vt-vm1 vm2 vm3"
+    variants:
+        - @default:
```

Comment:
I suggest not adding "@" for sub-case name, it hides the case name resulting in an inconvenient debug.

Reply:
Corrected